### ISSUE TITLE
Remove usage of `datetime_safe`

### ISSUE THREAD

**Issue by lociii** [justquick/django-activity-stream#524]
`django.utils.datetime_safe` has been deprecated and will be removed in DJango 5.0

[DJango source code](https://github.com/django/django/blob/stable/4.2.x/django/utils/datetime_safe.py) described it's purpose as following:

```
# These classes override date and datetime to ensure that strftime('%Y')
# returns four digits (with leading zeros) on years < 1000.
# https://bugs.python.org/issue13305
#
# Based on code submitted to comp.lang.python by Andrew Dalke
#
# >>> datetime_safe.date(10, 8, 2).strftime("%Y/%m/%d was a %A")
# '0010/08/02 was a Monday'
```

This may break for actions dated before the year 1000.  
If a removal without replacement is accepted, I would go ahead and provide a pull request.

**Comment by justquick** [comment]
anything throwing deprecation warnings for future django releases should be fine. please go ahead and make a PR

i dont think a lot of folks are using this app to track actions from the roman empire but it should b removed anyway

**Comment by lociii** [comment]
After some investigation, this seems like a packaging/release issue.

You already removed the `datetime_safe` calls in March 2022 in https://github.com/justquick/django-activity-stream/commit/df4b56904ccd540c71e38f3491e89dfd8b5caf93 and they are not found anymore in the `main` branch but the current package `1.4.2` still contains them in e.g. `actstream/feed.py:10`
```python
from django.utils import datetime_safe
```

Package has been downloaded from GitHub and PyPI both containing the outdated imports and usages.

**Comment by justquick** [comment]
fixed in version 1.4.3
